lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -132,6 +133,11 @@ def failure_lines
               encoded_string(exception.message.to_s).split("\n").each do |line|
                 lines << "  #{line}"
               end
+              if @extra_failure_lines.is_a?(Array) && !@extra_failure_lines.empty?
+                lines << ''
+                lines += @extra_failure_lines
```

Comment:
It's odd to mutate `lines` everywhere else in this function but on this one line to re-assign it to new array instance.  Instead, you can do:

``` ruby
@lines.concat(@extra_failure_lines)
```

...which continues to mutate it like on the other lines.